x-pack/plugins/enterprise_search/public/applications/app_search/components/result_settings/result_settings_table/non_text_fields_body.tsx

```diff
@@ -39,6 +40,10 @@ export const NonTextFieldsBody: React.FC = () => {
           </EuiTableRowCell>
           <EuiTableRowCellCheckbox>
             <EuiCheckbox
+              aria-label={i18n.translate(
+                'xpack.enterpriseSearch.appSearch.engine.resultSettings.table.rawArialLabel',
+                { defaultMessage: 'Toggle raw enabled' }
```

Comment:
Not a change request, just thinking out - can you leave out `enabled` from this label and have it still make sense? I was just wondering if screen readers would read out "Toggle raw enabled - disabled" and if that would be confusing. Maybe `Toggle raw field` or just `Toggle raw`?

Reply:
+1

Reply:
I appreciate the help: 894fd56b04ed6ac5451d6ddce11fc40709b57940